modules/git/blame.go

```diff
@@ -101,28 +106,44 @@ func (r *BlameReader) Close() error {
 	r.bufferedReader = nil
 	_ = r.reader.Close()
 	_ = r.output.Close()
+	if r.ignoreRevsFile != nil {
+		_ = util.Remove(r.ignoreRevsFile.Name())
+	}
```

Comment:
After `Remove`, maybe the ignore revs file should be set to empty, to avoid next Close call removes a file which doesn't belong to current BlameReader.

Reply:
The file has a random name which prevents this.